sdk/storage/Azure.Storage.Queues/src/Models/EncryptedMessageSerializer.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text;
+using System.Text.Json;
+using Azure.Storage.Cryptography.Models;
+
+namespace Azure.Storage.Queues.Specialized.Models
+{
+    internal static class EncryptedMessageSerializer
```

Comment:
Nit - Any reason not to put these in the `EncryptedMessage` class?

Reply:
Do you mean placing the serializer as a nested class to what it is serializing?

Reply:
No, just merging the two classes together.